### ISSUE TITLE
Double paste broken with the new paste menu

### ISSUE THREAD

**Issue by ferrybig** [microsoft/vscode#186343]
Type: <b>Bug</b>

My workflow to copy a line is "CTRL+C" "CTRL+V" "ctrl+V"

This worked in the lder vscode versions

In the new vscode versions this does not work at all, and it seems like it ignores any repeated paste events, until another input is done like arrow right then arrow left

Trying to disable the new behavior via `"editor.pasteAs.showPasteSelector": "never".` does not print back the pre May update version behavior

VS Code version: Code 1.79.2 (695af097c7bd098fbf017ce3ac85e09bbc5dda06, 2023-06-14T08:59:55.818Z)
OS version: Linux x64 6.3.7-arch1-1
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|AMD Ryzen 7 PRO 6850U with Radeon Graphics (16 x 3170)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>video_decode: enabled<br>video_encode: disabled_software<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|1, 1, 1|
|Memory (System)|30.13GB (19.89GB free)|
|Process Argv|--disable-smooth-scrolling --crash-reporter-id f959c43c-daf5-4b42-93c8-a032ec9ce4d9|
|Screen Reader|no|
|VM|0%|
|DESKTOP_SESSION|xfce|
|XDG_CURRENT_DESKTOP|XFCE|
|XDG_SESSION_DESKTOP|xfce|
|XDG_SESSION_TYPE|x11|
</details><details><summary>Extensions (46)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-sql-formatter|adp|1.4.4
vscode-css-modules|and|1.2.3
vscode-css-modules|cli|0.5.1
vscode-markdownlint|Dav|0.51.0
vscode-eslint|dba|2.4.2
chmod|dle|1.1.1
EditorConfig|Edi|0.16.4
vscode-typescript-exportallmodules|eli|2.6.0
prettier-vscode|esb|9.16.0
vscode-firefox-debug|fir|2.9.8
remotehub|Git|0.58.0
vscode-pull-request-github|Git|0.66.1
gitlab-workflow|Git|3.68.0
go|gol|0.39.0
css-modules-better-intellisense|Lim|0.0.3
caddyfile-support|mat|0.3.0
vscode-docker|ms-|1.25.1
remote-containers|ms-|0.295.0
remote-ssh|ms-|0.102.0
remote-ssh-edit|ms-|0.86.0
remote-wsl|ms-|0.79.5
vscode-remote-extensionpack|ms-|0.24.0
azure-repos|ms-|0.34.0
cmake-tools|ms-|1.14.33
cpptools|ms-|1.16.3
makefile-tools|ms-|0.7.0
remote-explorer|ms-|0.4.0
remote-repositories|ms-|0.36.0
remote-server|ms-|1.2.1
vsliveshare|ms-|1.0.5873
prisma|Pri|4.16.1
java|red|1.19.0
vscode-xml|red|0.25.0
LiveServer|rit|5.7.9
code-spell-checker|str|2.20.5
code-spell-checker-dutch|str|0.3.9
shellcheck|tim|0.32.6
intellicode-api-usage-examples|Vis|0.2.7
vscodeintellicode|Vis|1.2.30
vscode-java-debug|vsc|0.51.0
vscode-java-dependency|vsc|0.23.0
vscode-java-pack|vsc|0.25.11
vscode-java-test|vsc|0.39.0
vscode-maven|vsc|0.41.0
vscode-css-variables|vun|2.6.3
vitest-explorer|Zix|0.2.41

(2 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
vserr242cf:30382550
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
vscoreces:30445986
vscod805cf:30301675
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
vsaa593cf:30376535
pythonvs932:30410667
py29gd2263:30776702
vsclangdc:30486549
c4g48928:30535728
dsvsc012cf:30540253
pynewext54:30695312
azure-dev_surveyone:30548225
3biah626:30602489
pyind779:30671433
89544117:30613380
vscrp:30673768
pythonsymbol12:30671437
a9j8j154:30646983
showlangstatbar:30737416
vsctsb:30748421
pythonms35:30701012
03d35959:30757346
pythonfmttext:30731395
pythoncmv:30756943
fixshowwlkth:30771522
hideindicator:30766889
pythongtdpath:30769146
bgfeh915:30769767
e440d664:30776459
pythonidxpt:30772539
pythondjangots:30772535
pythonnocebcf:30776496
h7j2d465:30772216

```

</details>

**Comment by joyceerhl** [comment]
As a possible workaround try `"editor.pasteAs.enabled": false`?

**Comment by ferrybig** [comment]
> As a possible workaround try `"editor.pasteAs.enabled": false`?

That workaround the issue. The new pasteAs system still needs this bug squashed.

After setting it to false, if I open a workspace, wait for all extensions to activate, then open a .java file, I can now use CTRL+C followered by multiple CTRL+V to dublicat elines (where before it looked like the CTRL+V shortcut was broken) 

I did press "check for updates", (I run vscode on Arch, so updates need to be installed manually, I was a bit late with the May update) but it said there aren't any updates, so I cannot test your fix yet

**Comment by joyceerhl** [comment]
@ferrybig the fix is in our nightly Insiders build: https://code.visualstudio.com/insiders if you would like to download that and try it out (you can have stable and insiders side by side)

**Comment by mjbvz** [comment]
Fixed by #186649
